### PR TITLE
OSDOCS#7657: Adding new AWS permissions

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4.14+

Issue:
This issue addresses [osdocs-7657](https://issues.redhat.com/browse/OSDOCS-7657).

Link to docs preview:

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by mjpytlak** [comment]
Hi @yunjiang29 looks like we have to add a few permissions based on this conversation [1] with @barbacbd. PTAL. I see you were involved in the doc review when this topic was added [2]. Are you able to confirm if these permissions belong in the control plane list or compute list? Thanks in advance.

[1] https://redhat-internal.slack.com/archives/C68TNFWA2/p1693472847139919
[2] https://github.com/openshift/openshift-docs/pull/31702

**Review by barbacbd — COMMENTED** [review]
/lgtm

**Comment by barbacbd** [comment]
We should probably add @patrickdillon and QE

**Comment by mjpytlak** [comment]
> We should probably add @patrickdillon and QE

Thanks for the review @barbacbd. I had added @yunjiang29 to the original request.

File: modules/installation-aws-permissions-iam-roles.adoc [hunk 1]
```diff
@@ -20,6 +20,8 @@ The following lists specify the default permissions for control plane and comput
 * `ec2:DeleteSecurityGroup`
 * `ec2:DeleteVolume`
 * `ec2:Describe*`
+* `ec2:DescribeSecurityGroups`
+* `ec2:DescribeSecurityGroupRules`
```
[the remark this answers is not in the document]
Comment (yunjiang29) [reply modules/installation-aws-permissions-iam-roles.adoc:24]: > I just noticed that the previous line is a wildcard that should technically already include these two, so we may not actually need this.

+1

By default, `"ec2:Describe*",` already presents in the current control plane role policy.

**Comment by mjpytlak** [comment]
So good to close this PR @yunjiang29 or still looking into it?

**Comment by yunjiang29** [comment]
@mjpytlak I'm good to close this issue.

**Comment by mjpytlak** [comment]
No docs update required.

**Pull request closed without merging**